stories/assembly-saas-post-install-config.adoc

```diff
@@ -2,5 +2,9 @@ ifdef::context[:parent-context: {context}]
 
 [id="saas-post-install-config"]
 = Configuring {SaaSonAWS}
+After you have subscribed to {SaaSonAWSShort} and have access to {PlatformNameShort}, you must configure your automation mesh nodes, then set up your automation jobs. 
+
+For help with configuring your automation mesh see link:https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.4/html-single/red_hat_ansible_automation_platform_automation_mesh_for_operator-based_installations/index#idm140541758531360
```

Comment:
I'm not sure if we've cut over to using docs.redhat.com for the {BaseURL} attribute but it may be worth using to future proof the URL. Plus the {PlatformVers} attribute instead of 2.4.